app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -172,29 +172,33 @@ def parse_env_variables(env_variables)
       end
 
       def parse_openshift_image(openshift_image)
-        id = openshift_image[:dockerImageReference]
+        id = openshift_image[:dockerImageReference] || openshift_image[:metadata][:name] || ""
```

Comment:
I agree, Maybe we should abort the processing of that image in this case. There is not much meaning to an image without a name.